x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -200,52 +205,75 @@ alternative to role mapping.
+they connect to {kib},
+{kibana-ref}/kibana-authentication.html#pki-authentication[enable PKI authentication in {kib}].
+
 [[pki-realm-for-proxied-clients]]
 ==== PKI authentication for clients connecting to {kib}
 
 By default, the PKI realm relies on the node's network interface to perform the
 SSL/TLS handshake and extract the client certificate. This behaviour requires
-that that clients connect directly to {es} so that their SSL connection is
-terminated by the {es} node.  If SSL/TLS authenticatication is to be performed
-by {kib}, the PKI realm must be configured to permit delegation.
+that clients connect directly to {es} so that their SSL connection is terminated
+by the {es} node.  If SSL/TLS authentication is to be performed by {kib}, the
+PKI realm must be configured to permit delegation.
 
 Specifically, when clients presenting X.509 certificates connect to {kib},
 {kib} performs the SSL/TLS authentication. {kib} then forwards the client's
-certificate chain, by calling an {es} API, to have them further validated by
+certificate chain (by calling an {es} API) to have them further validated by
 the PKI realms that have been configured for delegation.
 
 To permit authentication delegation for a specific {es} PKI realm, start by
 configuring the realm for the usual case, as detailed in the
-<<pki-realm-for-direct-clients>>
-section. Note that you must explicitly configure a `truststore` (or,
-equivalently `certificate_authorities`) even though it is the same trust
-configuration that you have configured on the network layer.  Afterwards,
-simply toggle the `delegation.enabled` realm setting to `true`.  This realm is
-now allowed to validate delegated PKI authentication (after restarting {es}).
+<<pki-realm-for-direct-clients>> section. Note that you must explicitly
+configure a `truststore` (or, equivalently `certificate_authorities`) even
+though it is the same trust configuration that you have configured on the
+network layer. The `xpack.security.authc.token.enabled` setting must also be
+`true`. Then simply toggle the `delegation.enabled` realm setting to `true`. For example:
+
+[source, yaml]
+------------------------------------------------------------
+xpack:
+  security:
+    authc:
+      token.enabled: true
+      realms:
+        pki:
+          pki1:
+            order: 1
+            delegation.enabled: true
+            truststore:
+              path: "pki1_truststore.jks"
+------------------------------------------------------------
+
+After you restart {es}, this realm can validate delegated PKI authentication.
+For more information about these settings, see <<security-settings>>.
+
+[NOTE]
+====
 
-NOTE: PKI authentication delegation requires that the
-`xpack.security.authc.token.enabled` setting be `true` and that SSL/TLS be
-configured (without SSL/TLS client authentication).
+. PKI authentication delegation requires that SSL/TLS is configured (without
+SSL/TLS client authentication). See <<ssl-tls>>.
```

Comment:
> without SSL/TLS client authentication. See <<ssl-tls>>

This sentence is a bit unclear to me, since I don't think we describe a way to configure TLS without authentication in the linked page. Can we clarify or is there a better link?